native/src/main/java/io/ballerina/stdlib/persist/sql/datastore/SQLProcessor.java

```diff
@@ -131,16 +136,22 @@ static Object queryOne(Environment env, BObject client, BArray path, BTypedesc t
 
         Future balFuture = env.markAsync();
         env.getRuntime().invokeMethodAsyncSequentially(
-                getPersistClient(client, entity), Constants.RUN_READ_BY_KEY_QUERY_METHOD,
-                null, null, new Callback() {
+                // Call `SQLClient.runReadByKeyQuery(
+                //      typedesc<record {}> rowType, typedesc<record {}> rowTypeWithIdFields, anydata key,
+                //      string[] fields = [], string[] include = [], typedesc<record {}>[] typeDescriptions = []
+                // )`
+                // which returns `record {}|persist:Error`
```

Comment:
Do we need to keep these lines

Reply:
We're not skipping any lines. These are just comment s explaining which method is called